### PR TITLE
moving test hex light graphs here

### PR DESCRIPTION
**Pull request by dehann**
cc @Affie @GearsAD

### CODE REVIEW

**Comment by dehann** [comment]
was waiting on JuliaRobotics/IncrementalInference.jl#375

**Comment by dehann** [comment]
part of JuliaRobotics/IncrementalInference.jl#368

File: test/dfg/HexagonalCloud.jl [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+Gadfly.draw(Gadfly.PDF("/tmp/test2.pdf", 20cm, 10cm),pl)  # or PNG(...)
+
+
+# Add landmarks with Bearing range measurements
+p2br2 = Pose2Point2BearingRange(Normal(0,0.1),Normal(20.0,1.0))
+addFactor!(fg, [:x6; :l1], p2br2 )
+
+
+# solve
+batchSolve!(fg, drawpdf=true)
```
Comment (dehann) [inline test/dfg/HexagonalCloud.jl]: tree, smt, hist = solveTree!(fg)

File: test/dfg/HexagonalCloud.jl [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+
+# redraw
+pl = drawPosesLandms(fg, meanmax=:mean)
```
Comment (dehann) [inline test/dfg/HexagonalCloud.jl]: comment out drawing functions in tests

File: test/runtests.jl [hunk 3]
```diff
@@ -4,78 +4,92 @@
...
+@testset "Point2Point2 Init Tests" begin
 include("testPoint2Point2Init.jl")
+end
 
+@testset "BasicPose2Stationary Tests" begin
 include("testBasicPose2Stationary.jl")
+end
 
-println("[TEST] Pose2 evaluations...")
+@testset "Pose2 Tvaluations" begin
```
Comment (dehann) [inline test/runtests.jl]: "Evaluations"

File: test/runtests.jl [hunk 4]
```diff
@@ -4,78 +4,92 @@
...
+end
 
-@testset "[TEST] Pose2 evaluations..." begin
-  include("testDynPoint2D.jl")
+@testset "Pose2 evaluations..." begin
+include("testDynPoint2D.jl")
 end
 
+@testset "Point2Point2WorldBearing Tests" begin
 include("testPoint2Point2WorldBearing.jl")
```
Comment (dehann) [inline test/runtests.jl]: missing end

**Comment by dehann** [comment]
Does this need DFG v0.4?

File: test/dfg/HexagonalCloud.jl (hunk 1 above)
Comment (GearsAD) [reply test/dfg/HexagonalCloud.jl]: These tests aren't called yet, as they are I'll update them, just moving for the moment

File: test/dfg/HexagonalCloud.jl (hunk 2 above)
Comment (GearsAD) [reply test/dfg/HexagonalCloud.jl]: As above

File: test/runtests.jl (hunk 3 above)
Comment (GearsAD) [reply test/runtests.jl]: Cool, updated!

File: test/runtests.jl (hunk 4 above)
Comment (GearsAD) [reply test/runtests.jl]: Cool thanks!

**Comment by dehann** [comment]
requires JuliaRobotics/IncrementalInference.jl#380

**Comment by dehann** [comment]
can we do this as part of dfg v0.4 or do we need more stuff?  Basically can I merge this now?

**Comment by dehann** [comment]
HI All, just wanted to make sure we still wanted to add this to RoME?

**Comment by dehann** [comment]
Hi @GearsAD, could we maybe move these cloud tests to Caesar.jl instead?  RoME tests already taking quite long and maybe better have all integration type tests in Caesar.jl as the framework ... even though it can run in RoME.

**Pull request closed without merging**